Formula/aws-nuke.rb

```diff
@@ -0,0 +1,26 @@
+require "etc"
+class AwsNuke < Formula
+  desc "Nuke a whole AWS account and delete all its resources"
+  homepage "https://github.com/rebuy-de/aws-nuke"
+  url "https://github.com/rebuy-de/aws-nuke/archive/v2.8.0.tar.gz"
+  sha256 "3539303b864e983e7b8b3e43f83ae9a4900b8361170aac0cd123132f16d4acc4"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    srcpath = buildpath/"src/github.com/rebuy-de/aws-nuke"
+    srcpath.install buildpath.children
+
+    cd srcpath do
+      system "make", "build"
+      bin.install "aws-nuke--darwin-amd64" => "aws-nuke"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match "resource-types", shell_output("#{bin}/aws-nuke --help")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
so this is the tricky part, as this application's purpose is to completely delete all things within an aws account. apart from that, it doesn't have a purpose or any other functionality. I don't really see any self contained test that is possible here apart from the fact that the application itself doesn't error out when invoked.

Reply:
> If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected